src/rgw/rgw_admin.cc

```diff
@@ -2993,6 +2993,9 @@ int main(int argc, char **argv)
 	  cerr << "realm.rename failed: " << cpp_strerror(-ret) << std::endl;
 	  return -ret;
 	}
+        cout << "Realm name updated. Note that this change only applies to "
+            "the current cluster, so this command must be run separately "
```

Comment:
@smithfarm I believe this is incorrect sentence"so this command must be run separately" should be something as:
"so this command must run separately"